File: Proc-0/Writer.cs
/// <summary>
/// Писатель.
/// </summary>
interface Writer
{
	/// <summary>
	/// Запись ЕДИНИЦЫ информации из другого потока в текущий поток?
	/// Предположительно любая область памяти в Proc будет потоком-лентой.
	/// </summary>
	object Write(Reader source);

	object Write(object source, object baseobj); // TODO Временная заглушка для старых языков.
}

File: Proc-0/src/main/kotlin/Interop/Writer.kt
package proc.Interop

/**
 * Писатель.
 */
interface Writer {

	/**
	 * Запись ЕДИНИЦЫ информации из другого потока в текущий поток?
	 * Предположительно любая область памяти в Proc будет потоком-лентой.
	 */
	fun Write(source: Reader): Any

	fun Write(source: Any, baseObj: Any): Any // TODO Временная заглушка для старых языков.
}
